extension/src/extension.ts

```diff
@@ -241,6 +241,7 @@ export class Extension extends Disposable implements IExtension {
       this.config.onDidChangeExecutionDetails(async () => {
         const stopWatch = new StopWatch()
         try {
+          this.changeSetupStep()
```

Comment:
[F] Will update the details displayed on the Setup screen for where the CLI will be installed.